conf/test.config

```diff
@@ -20,9 +20,47 @@ params {
     max_time   = '6.h'
 
     // Input data
-    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = params.pipelines_testdata_base_path + 'viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
+    input = 'https://github.com/nf-core/test-datasets/raw/rangeland/Landsat_collection2/Landsat_data.tar.gz'
+    dem   = 'https://github.com/nf-core/test-datasets/raw/rangeland/dem/dem.tar.gz'
+    wvdb  = 'https://github.com/nf-core/test-datasets/raw/rangeland/wvp/wvdb.tar.gz'
 
-    
+    input_tar = true
+    dem_tar   = true
+    wvdb_tar  = true
+
+    data_cube = 'https://github.com/nf-core/test-datasets/raw/rangeland/datacube/datacube-definition.prj'
+    aoi       = 'https://github.com/nf-core/test-datasets/raw/rangeland/vector/aoi.gpkg'
+    endmember = 'https://github.com/nf-core/test-datasets/raw/rangeland/endmember/hostert-2003.txt'
+
+    // Remote sensing imagery parameters
+    start_date = '1987-01-01'
+    end_date   = '1989-12-31'
+
+    sensors_level1 = 'LT04,LT05'
+    sensors_level2 = 'LND04 LND05'
+
+    // Reference data
+    woody_change_ref = 'https://github.com/nf-core/test-datasets/raw/rangeland/reference/woody_cover_chg_ref.tif'
+    woody_yoc_ref    = 'https://github.com/nf-core/test-datasets/raw/rangeland/reference/woody_cover_yoc_ref.tif'
+
+    herbaceous_change_ref = 'https://github.com/nf-core/test-datasets/raw/rangeland/reference/herbaceous_cover_chg_ref.tif'
+    herbaceous_yoc_ref    = 'https://github.com/nf-core/test-datasets/raw/rangeland/reference/herbaceous_cover_yoc_ref.tif'
+
+    peak_change_ref = 'https://github.com/nf-core/test-datasets/raw/rangeland/reference/peak_chg_ref.tif'
+    peak_yoc_ref    = 'https://github.com/nf-core/test-datasets/raw/rangeland/reference/peak_yoc_ref.tif'
+
+    // Other parameters
+    group_size = 10
+
+    // enable mosaic for result checking
+    mosaic_visualization  = true
+
+    validationSchemaIgnoreParams = "peak_yoc_ref,peak_change_ref,herbaceous_yoc_ref,herbaceous_change_ref,woody_yoc_ref,woody_change_ref,config_profile_description,config_profile_name"
```

Comment:
The last two are indeed unnecessary to ignore. The reason why we ignore the reference parameters is because they are no 'standard' input to the pipeline and not defined in the schema. Executing the test profile without these will print:

```
WARN: The following invalid input values have been detected:

* --woody_change_ref: https://github.com/nf-core/test-datasets/raw/rangeland/reference/woody_cover_chg_ref.tif
* --woody_yoc_ref: https://github.com/nf-core/test-datasets/raw/rangeland/reference/woody_cover_yoc_ref.tif
* --herbaceous_change_ref: https://github.com/nf-core/test-datasets/raw/rangeland/reference/herbaceous_cover_chg_ref.tif
* --herbaceous_yoc_ref: https://github.com/nf-core/test-datasets/raw/rangeland/reference/herbaceous_cover_yoc_ref.tif
* --peak_change_ref: https://github.com/nf-core/test-datasets/raw/rangeland/reference/peak_chg_ref.tif
* --peak_yoc_ref: https://github.com/nf-core/test-datasets/raw/rangeland/reference/peak_yoc_ref.tif
```
Should i add these to the schema as hidden parameters?

Reply:
This was resolved by migrating to nf-test